ACE/ace/UNIX_Addr.cpp

```diff
@@ -51,9 +60,23 @@ ACE_UNIX_Addr::string_to_addr (const char addr[])
 int
 ACE_UNIX_Addr::addr_to_string (ACE_TCHAR s[], size_t len) const
 {
-  ACE_OS::strsncpy (s,
-                    ACE_TEXT_CHAR_TO_TCHAR (this->unix_addr_.sun_path),
-                    len);
+  if (!s || !len)
```

Comment:
I would prefer explicit numeric comparisons instead of implicit size_t to bool comparison